docs/unicef-bebbo-style-guide.md

```diff
@@ -0,0 +1,123 @@
+# Unicef Bebbo CMS Coding Style Guide
```

Comment:
Is this document identical with [Unicef Bebbo Coding Style Guide](https://docs.google.com/document/d/1g3qSJVisdVNMQLUDxQBPhwGcY1_4ZR0X/edit?usp=sharing&ouid=109061601149659036073&rtpof=true&sd=true)?

Reply:
Yes
Just that this one is in markdown format

Reply:
Can you also add readme and code of conduct to this PR?

Reply:
I have updated code of conduct. Let me check on readme demo.

Reply:
Loveth, CMS developer is very busy next few days it seems. Can you please go ahead with the readme commit too? He will update it later if required. Thanks.

Reply:
Updated the README. I update contributors mode of communication would happen of Github Discussion

Reply:
https://github.com/UNICEFECAR/parenting-app-bebbo-CMS/discussions